telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/bmh-node3-example.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: node3-example-credentials
+type: Opaque
+data:
+  username: ${BMC_NODE3_USERNAME}
+  password: ${BMC_NODE3_PASSWORD}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: controlplane-0-networkdata
+type: Opaque
+stringData:
+  networkData: |
+    interfaces:
+    - name: ${CONTROLPLANE3_INTERFACE}
+      type: ethernet
+      state: up
+      mac-address: "${CONTROLPLANE3_MAC}"
+      ipv4:
+        address:
+        - ip:  "${CONTROLPLANE3_IP}"
+          prefix-length: "${CONTROLPLANE3_PREFIX}"
+        enabled: true
+        dhcp: false
+    dns-resolver:
+      config:
+        server:
+        - "${DNS_SERVER}"
+    routes:
+      config:
+      - destination: 0.0.0.0/0
+        next-hop-address: "${CONTROLPLANE3_GATEWAY}"
+        next-hop-interface: ${CONTROLPLANE3_INTERFACE}
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: node3-example
+  labels:
+    cluster-role: control-plane
+spec:
+  online: true
+  bootMACAddress: ${BMC_NODE3_MAC}
+  bmc:
+    address: ${BMC_NODE3_ADDRESS}
+    disableCertificateVerification: true
+    credentialsName: node3-example-credentials
+  preprovisioningNetworkDataName: controlplane-0-networkdata
```

Comment:
Is this name the same for all three BMHs?

Reply:
changed